packages/react/src/components/FileUploader/FileUploader.js

```diff
@@ -99,6 +99,12 @@ export class FileUploaderButton extends Component {
      * Specify whether file input is disabled
      */
     disabled: PropTypes.bool,
+
+    /**
+     * Specify the size of the button, from a list of available sizes.
+     * For `default` buttons, this prop can remain unspecified.
+     */
+    size: PropTypes.oneOf(['default', 'field', 'small']),
```

Comment:
@tw15egan @laurenmrice Any thoughts on the list of size variant names? Just wanted to see if we have a chance to align this with some other components. Thanks!

Reply:
Text input, Number input, Dropdown and Combobox all currently have this format (Extra large size (XL), Default size, Small size (sm)). While Button has this format (Default, Field, Small). Both are inconsistent to describe the same thing. 

For file uploader, the drag and drop gets a knob for filename size, while the regular file uploader gets one knob for both button and a filename size because they need to scale together. I think for now it is probably fine how we have it, we are planning on reevaluating the naming convention of these components as apart of v11 to keep them all consistent, and that would fix this problem.

Reply:
We would eventually want it to be something like `large`, `medium / (default)`, and `small`

Reply:
I copied the `field` and `small` naming over from `Button` since the file uploader is using a button but I can change it to lg, md/default, and sm. would we want that instead?

Reply:
Yeah I know, I am thinking if we change this one to a new naming convention right now it may confuse people even more because then it is different from the current button component.

Reply:
got it, I will leave it for now